### DISCUSSION TITLE
Is there a good way to "shift" a color theme?

### DISCUSSION THREAD

**Discussion by IHIutch** [chakra-ui/chakra-ui#5190] [Q&A] [answered]
Say you're building a color scheme like this: 
```
export default {
  colors: {
   teal: {
      50: "#fafefd",
      100: "#f1fcfa",
      200: "#e7f9f5",
      300: "#d9f3ee",
      400: "#c7ebe5",
      500: "#afdfd7",
      600: "#8DCEC3",
      700: "#53b9ab",
      800: "#12a594",
      900: "#0e9888",
    },
  },
}
```

But I want my buttons and text to use `600, 700, 800` instead of `400, 500, 600`

<img width="426" alt="Screen Shot 2021-12-03 at 3 13 13 PM" src="https://user-images.githubusercontent.com/20825047/144667075-6a418401-2f26-4386-8c97-4e38ffa697c7.png">

Should I just move my hex values accordingly? (Although, now the tokens wouldn't really make sense anymore).
Or is there a way to do this within the theme itself?

**Comment by primos63** [comment, marked as answer]
The color levels are meant to convey the shade from lightest to darkest. Shifting the levels around will make reasoning about your app more difficult. I would argue for changing the colors via the component's theme. You mention buttons and text. I'm assuming that text refers to the Text component. It wouldn't make sense to me to style all text as it's either a dark gray in light mode and white in dark mode. If you do want to change the default text color, then I think global styles would be the place for that.

Here's a [CodeSandbox](https://codesandbox.io/s/custom-button-and-text-colors-levels-191es) with what I would do for changing color levels.

**Reply by IHIutch** [reply]
I think your example makes a lot of sense. 

I don't mean to suggest, for example, that overriding blue.500 from "#3182CE" to "#2C5282" is an appropriate solution (though I suppose you _could_ do that).

Nonetheless, I was thinking about this the wrong way, thanks for helping me understand.
